pkg/client/connector/service_proxy.go

```diff
@@ -188,6 +188,10 @@ func (p *mgrProxy) AgentLookupHostResponse(ctx context.Context, arg *managerrpc.
 	return client.AgentLookupHostResponse(ctx, arg, callOptions...)
 }
 
-func (p *mgrProxy) WatchLookupHost(info *managerrpc.SessionInfo, server managerrpc.Manager_WatchLookupHostServer) error {
+func (p *mgrProxy) WatchLookupHost(_ *managerrpc.SessionInfo, server managerrpc.Manager_WatchLookupHostServer) error {
 	return errors.New("WatchLookupHost is not implemented by the mgrProxy")
 }
+
+func (p *mgrProxy) WatchClusterInfo(_ *managerrpc.SessionInfo, server managerrpc.Manager_WatchClusterInfoServer) error {
+	return errors.New("WatchClusterInfo is not implemented by the mgrProxy")
+}
```

Comment:
I saw this and got real confused. "Won't the daemon need that!?"  I hadn't realized that the daemon hits the port-forward directly.  We should maybe be changing that.